fs-api/README.md

```diff
@@ -0,0 +1 @@
+# fs-api
```

Comment:
We should probably add some content here :) But can be done in a later PR as long as we don't forget (we could create an issue to prevent this)

Reply:
I'll add this to the follow-up PRs list in this PR's description, and create an issue for it once I've resolved all comments